src/frontend/src/components/ReactHookTextField.tsx

```diff
@@ -23,6 +23,7 @@ interface ReactHookTextFieldProps {
   errorMessage?: FieldError;
   maxLength?: number;
   required?: boolean;
+  readOnly?: boolean;
```

Comment:
it'd be better to add a 'disabled' prop that uses the 'disabled' prop on Textfield in this file